### DISCUSSION TITLE
Random sampling from unimodal ODF in the fundamental region

### DISCUSSION THREAD

**Discussion by xshang93** [mtex-toolbox/mtex#1455] [Ask Anything]
Hi, I wonder if you could kindly help me clear my confusion on what I am doing.

I'm trying to sample randomly from unimodal ODFs with different strengths of cubic texture. My code looks like:

```
csBeta = crystalSymmetry('432',[3.32 3.32 3.32],'mineral','Ti64 (beta)');
ss = specimenSymmetry('orthorhombic');
mod=orientation.byEuler(0,0,0,csBeta,ss);

odf_5 = unimodalODF(mod,'halfwidth',5*degree);
o_5 = discreteSample(odf_5,100);

odf_15 = unimodalODF(mod,'halfwidth',15*degree);
o_15 = discreteSample(odf_15,100);

odf_30 = unimodalODF(mod,'halfwidth',30*degree);
o_30 = discreteSample(odf_30,100);
```
When I tried to visualize the sampled orientations, I got:
<img width="487" alt="Screen Shot 2022-06-14 at 1 57 19 AM" src="https://user-images.githubusercontent.com/86078353/173503823-81105fd3-8b53-413a-bff3-5b87df8a331d.png">
which is pretty much what I'm expecting. 

However, I'm having a harding time understanding why the orientations are only found in the 1/4 corner of the fundamental region. I understand that this is due to the specimen symmetry being "orthorhombic", but how exactly is the specimen symmetry influencing this? I guess I might have some confusions on what a 'specimen' is referred to - is it my sample, or is it a grain cell? I feel this is a simple question but It would be great if you could help me understand it.

Thank you.

**Comment by kilir** [comment]
Hi,

I'd argue we live in a triclinic world, so does the specimen. As far as my understanding goes, applying specimen symmetry is used as a "shortcut" to cut down on on XRD measurement time when analysing e.g. a rolled material where top/bottom, left/right, front/rear would not differ since it is assumed that the imposed deformation results in a symmetric texture.

Cheers,
Rüdiger

**Reply by xshang93** [reply]
Thank you for the reply.
> 
> I'd argue we live in a triclinic world, so does the specimen. 
>
Is this due to the fact that in reality most of (if not all) the systems do not have a perfectly 90 degree angle?

> As far as my understanding goes, applying specimen symmetry is used as a "shortcut" to cut down on on XRD measurement time when analysing e.g. a rolled material where top/bottom, left/right, front/rear would not differ since it is assumed that the imposed deformation results in a symmetric texture.
> 
Thanks for pointing this out. In my case I'm only sampling a RVE so a symmetric texture doesn't work. But for a real life sample, where the RVEs are repetitively arranged, I guess using a symmetry does make a lot of sense.

**Reply by kilir** [reply]
> Is this due to the fact that in reality most of (if not all) the systems do not have a perfectly 90 degree angle?

What I meant is that north, east, south,west, heaven and hell can usually be distinguished, even some share angles of 90°, no symmetry can be applied.

With respect to your plan, I'd think it sounds like `specimenSymmetry('1')`, which is the same as simply not specifying a specimen Symmetry in mtex, is what you are looking for.

Hope that helps.

Cheers,
Rüdiger

**Reply by xshang93** [reply]
> What I meant is that north, east, south,west, heaven and hell can usually be distinguished, even some share angles of 90°, no symmetry can be applied.
> 
> With respect to your plan, I'd think it sounds like `specimenSymmetry('1')`, which is the same as simply not specifying a specimen Symmetry in mtex, is what you are looking for.
>

Thanks. What difference would that make, between '1' and 'triclinic'? My understanding is that since there is no symmetry involved within a triclinic system, shouldn't there be no difference from having no symmetry?

**Reply by kilir** [reply]
`'1'` means triclinic, the only symmetry involved is the identity.

**Reply by xshang93** [reply]
> `'1'` means triclinic, the only symmetry involved is the identity.

Of course! How did I miss that - My apologies. That sounds about right!

Thanks so much for your help
